src/lib/support/DefaultStorageKeyAllocator.h

```diff
@@ -197,6 +197,16 @@ class DefaultStorageKeyAllocator
         return StorageKeyName::Formatted("g/su/%x", static_cast<unsigned>(index));
     }
     static StorageKeyName SubscriptionResumptionMaxCount() { return StorageKeyName::Formatted("g/sum"); }
+    // Scene Storage
+    static StorageKeyName SceneFabricList()
+    {
+        return StorageKeyName::FromConst("g/gfl");
```

Comment:
This clashes with `static StorageKeyName GroupFabricList() { return StorageKeyName::FromConst("g/gfl"); }`. Is it the same? If so, then  don't create a new getter.

Reply:
This was meant to be the same to avoid doubling space in flash for the same list. Initially it was replaced by FabricList(), but reverted back because I had changed the "g/gfl", which would break OTA updates. I will revert this to "static StorageKeyName FabricList() { return StorageKeyName::FromConst("g/gfl");", thus preserving the key and remove "SceneFabricList".